config/env.development

```diff
@@ -187,22 +187,6 @@ DEPENDENCY_DISCOVERY_URL=http://localhost/v1/dependency-discovery
 # NODE_ENV should be one of "development" or "production"
 NODE_ENV=development
 
-# PORT is the port used by the web server
-PORT=3000
-
-# API_URL is the URL of the Telescope Web API server.  Change this to
-# pick which backend server our frontend uses.  If you are developing
-# locally, this will be localhost:{PORT}, probably http://localhost:3000.
-# Other possible values include:
-#
-#   (local development)
-#   API_URL=http://localhost:3000
-#   (our staging server)
-#   API_URL=https://dev.telescope.cdot.systems
-#   (our production server)
-#   API_URL=https://telescope.cdot.systems
-API_URL=http://localhost:3000
-
 # PROXY_FRONTEND=1 will allow proxying the Next dev server (http://localhost:8000)
```

Comment:
You can get rid of this next block too (`PROXY_FRONTEND=`) is only used in the backend.